src/components/MapSubtitle.vue

```diff
@@ -17,8 +17,8 @@
             regions across the continental U.S. Map shading indicates the natural 
             water storage relative to historical conditions for this time of year.
           </p>
-          <a href="https://www.google.com/">
-          <button>Learn More</button>
```

Comment:
I thought it was a pretty clever joke. We give them a link 'to learn more'. Then the link goes to a blank google page. LOL, it was like 'here sucker, look it up yourself'. Made me laugh.